Fixes blocking changes for mkdocs site migration (#46)
cbrownstein-lambda/lambda-docs-mkdocs: docs/public-cloud/1-click-clusters/support.md

---
description: How to access 1-Click Clusters
tags:
  - 1-click-clusters
  - support
---

# Support

> At Lambda, we recognize that exceptional support is critical to maximizing the value of your One-Click Cluster (1CC) deployment. Our world-class support team is dedicated to ensuring your success at every stage, from deployment to daily operations.

Reviewer: ```suggestion
At Lambda, we recognize that exceptional support is critical to maximizing the value of your 1-Click Cluster (1CC) deployment. Our world-class support team is dedicated to ensuring your success at every stage, from deployment to daily operations.
```

## Support team

When you choose Lambda, you gain access to a dedicated support team of seasoned professionals with deep expertise in AI/ML infrastructure. This team includes:

> * **Customer Success Manager (CSM)**: Your main point of contact post sales, responsible for ensuring your overall satisfaction and delivery of your solution.

Reviewer: ```suggestion
* **Customer Success Manager (CSM)**: Your main point of contact post-sales, responsible for ensuring the delivery of your solution and your overall satisfaction.
```

> * **Technical Account Manager(TAM)**: An expert specialist who understands the intricacies of your deployment, provides ongoing technical guidance, and escalates any complex issues. 

Reviewer: ```suggestion
* **Technical Account Manager(TAM)**: An expert specialist who understands the specifics of your deployment, provides ongoing technical guidance, and escalates any complex issues. 
```

> * **Machine learning expert (MLE)**: A AI/ML expert within lambda will provide guidance how how to integrate and scale AI workloads.    

Reviewer: ```suggestion
* **Machine Learning Expert (MLE)**: An AI/ML expert within Lambda will provide guidance on how to integrate and scale AI workloads.    
```

> * **Support Engineering**: Lambda’s Support team, available 24/7 via our ticketing portal, is well-versed in 1CC and will be able to respond to any technical support request, escalating issues and incidents early and often.

Reviewer: ```suggestion
* **Support Engineering**: Lambda’s Support team, available 24/7 through our ticketing portal, is well-versed in 1CC and will be able to respond to any technical support request, escalating issues and incidents early and often.
```

> ## Support Scope

Reviewer: ```suggestion
## Support scope
```

Lambda classifies incoming support tickets into three categories:

* In scope
* Best effort
* Out of scope

> ### In Scope

Reviewer: ```suggestion
### In scope
```

* **Hardware and Infrastructure:** Full support for CPU/GPU VMs, physical hosts, and networking components.  

> * **Software Environment:** Assistance with Lambda Stack, OFED drivers, Jupyter Notebooks, and essential ML tools like NCCL and OpenMPI.  

Reviewer: ```suggestion
* **Software Environment:** Assistance with Lambda Stack, OFED drivers, Jupyter Notebooks, and essential ML tools like NCCL and Open MPI.  
```

* **Networking and Storage:** Management and troubleshooting for Ethernet and InfiniBand networks, as well as persistent and local storage.  
* **Slurm Installation:** Guidance on Slurm setup and basic troubleshooting to streamline your job scheduling processes.  

> * **Managed Kubernetes*:** If purchased as an add-on, our team of Kubernetes experts will be there to help you.

Reviewer: Are we sure we want to word it this way? It seems to promise a level of support that we don't necessarily want to be promising.

Author: This language has been blessed by Matt Hammons.

> ### Best Effort Support:

Reviewer: ```suggestion
### Best effort support:
```
